Pod/Classes/ios/NYTPhotosViewController.m

```diff
@@ -53,6 +53,13 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder NS_DESIGNATED_INITIALIZER;
 @property (nonatomic, readonly) UIView *referenceViewForCurrentPhoto;
 @property (nonatomic, readonly) CGPoint boundsCenterPoint;
 
+/**
+ *  Tracks whether the current dismissal of this view controller is due to user interaction
+ *  (an interactive pan or a tap on the Close button).
+ *  Reset to `NO` when an interactive dismissal is not in progress.
+ */
+@property (nonatomic, getter=isCurrentDismissalUserTriggered) BOOL currentDismissalUserTriggered;
```

Comment:
Could you potentially eliminate this intermediate state if both `doneButtonTapped:` and `didPanWithGestureRecognizer` called through to a common function, hypothetically something like `dismissViewControllerAnimated:isUserInitiated:completion:`, passing `YES` for `isUserInitiated`, while `dismissViewControllerAnimated:completion:` calls through to the common method with `NO`?